src/dispatch_common.c

```diff
@@ -478,6 +478,7 @@ epoxy_internal_has_gl_extension(const char *ext, bool invalid_op_mode)
 
         for (i = 0; i < num_extensions; i++) {
             const char *gl_ext = (const char *)glGetStringi(GL_EXTENSIONS, i);
+            if(! gl_ext) return false;
```

Comment:
It would be great if you could respect the coding style, and add a newline break for `return false`.